share/mk/src.opts.mk

```diff
@@ -318,9 +318,6 @@ BROKEN_OPTIONS+=LIB32
 # LIB64 is supported on aarch64*c* and riscv64*c*
 .if ${__T:Maarch64*c*} || ${__T:Mriscv64*c*}
 __DEFAULT_YES_OPTIONS+=LIB64
-# In principle, LIB32 could work on architectures where it's supported, but
-# Makefile.libcompat only supports one compat layer.
-BROKEN_OPTIONS+=LIB32
```

Comment:
Once aarch64 grows lib32 support I suspect we'll want to build releases with WITHOUT_LIB32 since Morello hardware doesn't support it and we don't want someone to find libraries and waste their time trying to get it to work (I'm also 100% certain the kernel side doesn't work even if it were to manage to compile.)